creator-node/scripts/verifyHealthCheckFileUpload.js

```diff
@@ -0,0 +1,73 @@
+const crypto = require('crypto')
+const axios = require('axios')
+const FormData = require('form-data')
+
+const { generateTimestampAndSignature } = require('../src/apiSigning')
+const { promisify } = require('util')
+
+const PRIVATE_KEY = process.env.delegatePrivateKey
+const CREATOR_NODE_ENDPOINT = process.env.creatorNodeEndpoint
+const randomBytes = promisify(crypto.randomBytes)
+
+/**
+ * Process command line args and issue file upload health check
+ */
+async function run () {
+  try {
+    parseEnvVarsAndArgs()
+  } catch (e) {
+    console.error(`\nIncorrect script usage: ${e.message}`)
+    console.error(`Script usage: node verifyHealthCheckFileUpload.js`)
+    return
+  }
+
+  try {
+    // Generate signature using local key
+    const randomBytesToSign = (await randomBytes(18)).toString()
+    const signedLocalData = generateTimestampAndSignature({ randomBytesToSign }, PRIVATE_KEY)
+    // Add randomBytes to outgoing request parameters
+    const reqParam = signedLocalData
+    reqParam.randomBytes = randomBytesToSign
+
+    let sampleTrack = new FormData()
+    sampleTrack.append('file', (await axios({
+      method: 'get',
+      url: 'https://s3-us-west-1.amazonaws.com/cheran.audius.co/97mb_music.mp3', // 97 MB
```

Comment:
where did you get this file? we should make sure it's clear to host like this. we should also move this to a bucket intended to store public files

Reply:
Sorry about this, I wasn't sure where to host the file, so I kept it on a personal s3 bucket for now.